docs/src/pages/premium-themes/onepirate/modules/components/TextField.tsx

```diff
@@ -65,7 +68,8 @@ const styles = (theme: any) =>
     },
   });
 
-export interface OnePirateTextFieldProps extends Omit<TextFieldProps, 'size'> {
+export interface OnePirateTextFieldProps
+  extends Omit<FilledTextFieldProps | StandardTextFieldProps, 'size'> {
```

Comment:
Thanks for fixing this. One thing I don't quite follow here --

`disableUnderline` is defined in [FilledInputProps](https://github.com/mui-org/material-ui/blob/afac5cd043d145d2d4af8066d67f6d98b8627dfe/packages/material-ui/src/FilledInput/FilledInput.d.ts#L47) which is used in [FilledTextFieldProps](https://github.com/mui-org/material-ui/blob/afac5cd043d145d2d4af8066d67f6d98b8627dfe/packages/material-ui/src/TextField/TextField.d.ts#L173) but not StandardTextFieldProps.

So why is the type here `Omit<FilledTextFieldProps | StandardTextFieldProps, 'size'>` instead of just `Omit<FilledTextFieldProps, 'size'>`?

Reply:
Ah n/m answered my own question, it's also defined in [InputProps](https://github.com/mui-org/material-ui/blob/afac5cd043d145d2d4af8066d67f6d98b8627dfe/packages/material-ui/src/Input/Input.d.ts#L41) which is used by StandardTextFieldProps.